addons/controllers/vspherecsiconfig_controller_test.go

```diff
@@ -274,26 +274,34 @@ var _ = Describe("VSphereCSIConfig Reconciler", func() {
 	})
 
 	Context("reconcile VSphereCSIConfig manifests in paravirtual mode", func() {
-		var availabilityZone *topologyv1alpha1.AvailabilityZone
+		var availabilityZone1, availabilityZone2 *topologyv1alpha1.AvailabilityZone
 		BeforeEach(func() {
 			// (deliberate decision): There is no watch on AvailabilityZone in vspherecsiconfig_controller so any change to it will not trigger reconcile
 			// of resources. Based on discussions with TKGS team, availability zone is created at supervisor cluster init time
 			// and does not really change after that. The test will only check for the presence of non legacy availability
 			// zone and set zone to true
-			availabilityZone = &topologyv1alpha1.AvailabilityZone{
+			availabilityZone1 = &topologyv1alpha1.AvailabilityZone{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "test-az",
 				},
 			}
 
-			Expect(k8sClient.Create(ctx, availabilityZone)).Should(Succeed())
+			availabilityZone1 = &topologyv1alpha1.AvailabilityZone{
```

Comment:
did you mean ```availabilityZone2``` here instead?

Reply:
i just fixed this in the latest push. can you please check